t/t1034-mf-priority-max-nodes.t

```diff
@@ -0,0 +1,103 @@
+#!/bin/bash
+
+test_description='test tracking node counts for associations in the priority plugin'
+
+. `dirname $0`/sharness.sh
+
+mkdir -p conf.d
+
+MULTI_FACTOR_PRIORITY=${FLUX_BUILD_DIR}/src/plugins/.libs/mf_priority.so
+SUBMIT_AS=${SHARNESS_TEST_SRCDIR}/scripts/submit_as.py
+DB_PATH=$(pwd)/FluxAccountingTest.db
+
+export TEST_UNDER_FLUX_SCHED_SIMPLE_MODE="limited=1"
+test_under_flux 5 job -o,--config-path=$(pwd)/conf.d
+
+flux setattr log-stderr-level 1
+
```

Comment:
Note: Due to changes in flux-core you'll now have to enable guest access to the testexec implentation with
```
flux config load <<-EOF
[exec.testexec]
allow-guests = true
EOF
```